python/ray/tune/trainable.py

```diff
@@ -330,6 +328,10 @@ def restore(self, checkpoint_path):
         self._timesteps_since_restore = 0
         self._iterations_since_restore = 0
         self._restored = True
+        logger.info("Restored from checkpoint: {}".format(self._experiment_id, checkpoint_path))
+        logger.info(
+            "Current state after restoring: done iterations {}, done timesteps {}, passed time {}, done episodes {}".format(
```

Comment:
I think it would be easier to debug if the strings are consistent with variable names. E.g.
```python
logger.info(
    "Current state after restoring: _iteration={}, _timesteps_total={}, _time_total={}, _episodes_total={}".format(
    self._iteration, self._timesteps_total, self._time_total, self._episodes_total))
```